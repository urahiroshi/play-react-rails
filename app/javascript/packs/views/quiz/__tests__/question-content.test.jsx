import React from 'react'
import renderer from 'react-test-renderer'
import QuestionContent from '../question-content'

const createComponent = () => (
  <QuestionContent
    page={1}
    content="<div>hoge</div>"
  />
)

describe('#render', () => {
  it('should show snapshot', () => {
    const component = renderer.create(createComponent())
    expect(component.toJSON()).toMatchSnapshot()
  })
})
