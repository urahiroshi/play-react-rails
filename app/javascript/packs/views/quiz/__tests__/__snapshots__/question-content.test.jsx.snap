// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render should show snapshot 1`] = `
<div
  className="form-group"
>
  <div
    className="col-sm-1 control-label quiz-header"
  >
    Q
    1
    .
  </div>
  <div
    className="col-sm-11 quiz-content"
  >
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": "<div>hoge</div>",
        }
      }
    />
  </div>
</div>
`;
